Services.md

```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
+- **/groups/api/v1/groups/{groupName}** : PUT : updates a group
+- **/groups/api/v1/groups/{groupName}** : DELETE : deletes a group
+- **/groups/api/v1/groups/{groupName}/details** : GET : detailed group information
+- **/groups/api/v1/terms/{groupName}** : GET : group terms
+- **/groups/api/v1/terms/{groupName}** : DELETE : delete group terms
+- **/groups/api/v1/terms/{groupName}** : POST : create group terms
+- **/groups/api/v1/terms/{groupName}** : PUT : update group terms
+- **/groups/api/v1/requests/{groupName}** : GET : group invitation requests
+- **/groups/api/v1/requests/{groupName}/{userUuid}** : reject an invitation request
+- **/groups/api/v1/invitations/{groupName}** : GET : group invitations
+- **/groups/api/v1/invitations/{groupName}** : POST : invite a member
+- **/groups/api/v1/invitations/{groupName}/email** : GET : group invitation email copy
+- **/groups/api/v1/invitations/{groupName}/email** : POST : set group invitation email copy
+- **/groups/api/v1/invitations/{groupName}/{userUuid}** : PUT : update an invitation
+- **/groups/api/v1/invitations/{groupName}/{userUuid}** : DELETE : delete an invitation
+- **/groups/api/v1/members/{groupName}** : GET : paginated members
+- **/groups/api/v1/members/{groupName}/{memberUuid}** : DELETE : delete a member
+- **/groups/api/v1/members/{groupName}/{memberUuid}/renew** : POST : renew a member
+- **/groups/api/v1/curators/{groupName}** : POST : add a curator
+- **/groups/api/v1/curators/{groupName}/{memberUuid}/downgrade** : POST : downgrade a curator to a normal member
+- **/groups/api/v1/users** : GET : search for users
+- **/groups/api/v1/users/any** : GET : search for any users
+- **/groups/api/v1/self/{groupName}** : DELETE : leave a group
+- **/groups/api/v1/self/requests** : GET : invitation requests from the logged in user
+- **/groups/api/v1/self/requests/{groupName}** : POST : request to be invited
+- **/groups/api/v1/self/requests/{groupName}** : DELETE : cancel a request to be invited
+- **/groups/api/v1/self/invitations** : GET : invitations for the logged in user
+- **/groups/api/v1/self/invitations/{groupName}** : POST : join a group
+- **/groups/api/v1/self/invitations/{groupName}** : DELETE : reject an invitation to a group
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add a member
+- **/groups/api/v1/sudo/logs/all/raw** : GET : returns an audit log containing records of all actions taken on groups over time
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : POST : subscribe user to nda mailing list
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : DELETE : unsubscribe user to nda mailing list
+- **/groups/api/v1/sudo/curators/{groupName}** : GET : curators for a given group
+- **/groups/api/v1/sudo/curators/{groupName}** : POST : add a curator to a given group
+- **/groups/api/v1/sudo/user/{userUuid}** : DELETE : delete a user
+- **/groups/api/v1/sudo/user/inactive** : DELETE : delete inactive users
+- **/groups/api/v1/sudo/user/consolidate** : DELETE :
```

Comment:
Sends all group members ships to CIS to consolidate (note: this treats DinoPark as source of truth).